packages/terra-doc-template/tests/jest/DocTemplate.test.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+import DocTemplate from '../../src/DocTemplate';
+import readme from '../../examples/test-examples/TestReadme.md';
+// eslint-disable-next-line import/no-duplicates
+import TestComponentExample from '../../examples/test-examples/TestComponentExample';
+// Had to remove !rawloader! from the next two imports as it broke jest, but in normal use it should be there
+// eslint-disable-next-line import/no-webpack-loader-syntax, import/first, import/no-unresolved, import/extensions, import/no-duplicates
+import exampleSrc from '../../examples/test-examples/TestComponentExample.jsx';
+// eslint-disable-next-line import/no-webpack-loader-syntax, import/first, import/no-unresolved, import/extensions
+import testComponentSrc from '../../examples/test-examples/TestComponent.jsx';
+
+jest.mock('../../examples/test-examples/TestReadme.md', () => 'test-file-stub');
```

Comment:
Since we are stubbing out these files, we could create a single test constant for the jest tests so we could drop the multiple imports above.